Alloy/MPS+Functors.swift

```diff
@@ -0,0 +1,29 @@
+import MetalPerformanceShaders
+
+public extension MPSUnaryImageKernel {
+    func callAsFunction(sourceTexture: MTLTexture,
+                        destinationTexture: MTLTexture,
+                        in commandBuffer: MTLCommandBuffer) {
+        self.encode(commandBuffer: commandBuffer,
+                    sourceTexture: sourceTexture,
+                    destinationTexture: destinationTexture)
+    }
+
+    func callAsFunction(inPlaceTexture: MTLTexture,
+                        fallbackCopyAllocator: MPSCopyAllocator? = nil,
+                        in commandBuffer: MTLCommandBuffer) {
+        var inPlaceTexture = inPlaceTexture
+        self.encode(commandBuffer: commandBuffer,
+                    inPlaceTexture: &inPlaceTexture,
```

Comment:
this is incorrect, it won't work for the cases when MPS kernel replaces the instance. Let's remove this functor or move to pointer argument instead